package.json

```diff
@@ -235,12 +221,16 @@
     "md5-file": "^5.0.0",
     "mobx": "^5.15.7",
     "mobx-observable-history": "^1.0.3",
+    "mobx-react": "^6.2.2",
     "mock-fs": "^4.12.0",
     "node-pty": "^0.9.0",
     "npm": "^6.14.8",
     "openid-client": "^3.15.2",
     "path-to-regexp": "^6.1.0",
     "proper-lockfile": "^4.1.1",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-router": "^5.2.0",
```

Comment:
Imported in both main/rendrerer runtimes, see: `page-registry.ts`
NB: `react-router` required implicitly by `react` or `react-dom`